caiman_napari/algorithms/cnmf.py

```diff
@@ -15,11 +15,10 @@
 
 from caiman.utils.visualization import get_contours as caiman_get_contours
 from caiman.source_extraction.cnmf.cnmf import load_CNMF
-from .utils import *
+from caiman_napari.utils import *
```

Comment:
Try replacing this with `from ..utils import *` and see if it fixes the import issues??

Reply:
Doesn't work unfortunately. WIll try to solve after I finish other changes you recommended